### PR TITLE
Provide backend address pool to LB and provide LB association for Ser…

### PR DESCRIPTION
**Pull request by JargoonPard**
…verGroup

### CODE REVIEW

**Comment by JargoonPard** [comment]
PTAL

File: clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/templates/AzureLoadBalancerResourceTemplate.groovy [hunk 1]
```diff
@@ -148,6 +152,15 @@ class AzureLoadBalancerResourceTemplate {
     }
   }
 
+  static class BackEndAddressPool{
+    String name;
```
Comment (duftler) [inline clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/templates/AzureLoadBalancerResourceTemplate.groovy]: Can drop the semicolon here.

Comment (duftler) [inline clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/templates/AzureLoadBalancerResourceTemplate.groovy]: Actually, throughout this file.

**Comment by duftler** [comment]
LGTM after addressing one comment and rebasing.

**Pull request merged**